### ISSUE TITLE
[IOS] Avatars are missing on Profiles on device screen after user did a logout or after a background

### ISSUE THREAD

**Issue by mariia-skrypnyk** [status-im/status-react#20016]
# Bug Report

### Reproduction

1. Install the app
2. Create a 2-3 users
3. Make a logout

#### Expected behavior
Design is [here](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=4357-629321&t=XHtWB6mMdqyJCC7z-0).

<img width="256" alt="Screenshot 2024-05-14 at 15 16 56" src="https://github.com/status-im/status-mobile/assets/147824399/92d2bd72-984b-4b17-a9db-e49c3cb4c54d">

#### Actual behavior
Avatar is missing

<img width="200" alt="Screenshot 2024-05-14 at 15 18 09" src="https://github.com/status-im/status-mobile/assets/147824399/56d4115c-5424-4282-a926-1c8f90e06489">

### Additional Information

- Status version: nightly 14/05
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

**Comment by churik** [comment]
So far for me it is reproducible mostly on IOS and every time after device was locked due to inactivity

Steps:
1) Create several profiles
2) Put Status to background
3) Wait until the device will be locked due to inactivity (5 mins in my case)

After opening the app again:

https://github.com/status-im/status-mobile/assets/4557972/31ecfe3c-1351-4db5-96e0-5452677efb9e

[logs (91).zip](https://github.com/user-attachments/files/15872659/logs.91.zip)

There is no other way to see avatars again until full app reopening,

**Comment by churik** [comment]
@mariia-skrypnyk can you please verify on your device is https://github.com/status-im/status-mobile/issues/20016#issue-2295446602 is still reproducible?

**Comment by Parveshdhull** [comment]
Update: I cannot reproduce this issue on the IOS simulator debug app. I am removing my assignment so someone who has a physical device setup can pick up this issue.

**Comment by churik** [comment]
postponed due to limited capacity for 2.30

**Comment by ilmotta** [comment]
> So far for me it is reproducible mostly on IOS and every time after device was locked due to inactivity

Precisely. In an iPhone I can reproduce most of the time by simply locking the device after logging out. I also noticed many times the avatars lose their custom color and ring, but then sometimes come back after locking and unlocking the device. This makes me think the cause should be associated with the media server because every time a device is locked the media server port changes.

This bug is likely related to this one https://github.com/status-im/status-mobile/issues/21203

**Comment by churik** [comment]
Note:
@ilmotta was able to reproduce it with his IOS device. 
It is reproducible for test team also quite a lot. 

Including in 2.32 with low-prio

**Comment by churik** [comment]
Just an asumption: might be related to https://github.com/status-im/status-mobile/issues/20982

**Comment by churik** [comment]
I don't think that we have a capacity for it in 2.32, given the desired release cut date.
Deferring the issue, please be free to move it bak if you think there is a room for it @ilmotta

**Comment by churik** [comment]
This one we haven't seen for quite a while, so closing it for now

**Comment by pavloburykh** [comment]
> This one we haven't seen for quite a while, so closing it for now

Have faced it today (Jan 30, 2025) in IOS nightly. Re-opening the issue. Looks like issue somehow related to app being in background for some time. Needs to be investigated. 

[logs (55).zip](https://github.com/user-attachments/files/18605883/logs.55.zip)
![Image](https://github.com/user-attachments/assets/fa6b18ec-5f80-4596-bf3d-47442e650fb6)
